package.json

```diff
@@ -23,7 +23,8 @@
     "start:demo": "yarn build:demo && live-server demo",
     "tag": "utils/scripts/version-next.sh",
     "test": "yarn test:all --watch",
-    "test:all": "jest --config=utils/test/jest.config.js"
+    "test:all": "jest --config=utils/test/jest.config.js",
+    "version": "yarn run build && git add -A"
```

Comment:
Since we inject the version number into our bundles this makes our bundle size change _after_ we version the package. This lifecycle hook ensures that all update `.size-snapshot` files are included.

Reply:
I'm not exactly following your comment. But this looks like it is going to add CI ⌛️ to the publish build process. Can we get rid of `version` by increasing the snapshot threshold so that the difference between `PACKAGE_VERSION` and actual version isn't considered to be a problem?

Reply:
This is only ran locally when we call yarn run tag. It shouldn't be ran in CI since we're only calling lerna publish.

Adding a threshold would allow the build to pass, but it still creates new .size-snapshot files which would dirty the next commit someone has to make.

Reply:
Makes sense. Thanks for the clarification.